screen/.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 11
+    },
+    "rules": {
+        "quotes": ["error", "double"],
```

Comment:
If I remember correctly, most of us preferred single quotes, right? I'm fine with switching to single quotes in UI if we want to be consistent. (I'm not sure what we said about consistency.)

Reply:
We can switch to single ^^

Reply:
Reminder to switch to single if you want, but I'll approve anyway.